File: abc/abc083/a.cpp
// https://atcoder.jp/contests/abc083/submissions/30402150

#include <bits/stdc++.h>
#define all(x) (x).begin(), (x).end()
using namespace std;
typedef long long ll;

int main() {
  cin.tie(nullptr);
  ios_base::sync_with_stdio(false);

  int a, b, c, d; cin >> a >> b >> c >> d;
  a += b;
  c += d;
  if(a > c) cout << "Left";
  else if(a < c) cout << "Right";
  else cout << "Balanced";
}

File: abc/abc083/a.java
// https://atcoder.jp/contests/abc083/submissions/41617938

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    int c = stdIn.nextInt();
    int d = stdIn.nextInt();
    
    int l = a + b;
    int r = c + d;

    if(l > r) {
      System.out.println("Left");
    } else if(l < r) {
      System.out.println("Right");
    } else {
      System.out.println("Balanced");
    }
  }
}
